tendrl/commons/utils/log_utils.py

```diff
@@ -0,0 +1,31 @@
+""" Handles logging functionality. """
+import sys
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+
+
+def log(log_priority,publisher_id,**log_payload):
```

Comment:
problem is when we call message call class with some value at that time caller is not required in __init__ param, but when message is received by handler via socket then it will validated using same message class, at that time again __init__ function is called from from_json  function at that time caller have some value, __inti__ function should have caller has parameter. so i gave that as optional.

now you are passing caller details from your function so it will always come so you all change remove that option thing.